src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/query-time-range.mdx

```diff
@@ -0,0 +1,2701 @@
+    </tr>
+    <tr>
+      <td>`wednesday at '01:23:45'`</td>
+      <td style="font-family:monospace">2023-12-**13T01:23:45.000**Z</td>
+    </tr>
+    <tr>
+      <td>`thursday`</td>
+      <td style="font-family:monospace">2023-12-**14T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`thursday at '01:23:45'`</td>
+      <td style="font-family:monospace">2023-12-**14T01:23:45.000**Z</td>
+    </tr>
+    <tr>
+      <td>`friday`</td>
+      <td style="font-family:monospace">2023-12-**15T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`friday at '01:23:45'`</td>
+      <td style="font-family:monospace">2023-12-**15T01:23:45.000**Z</td>
+    </tr>
+    <tr>
+      <td>`saturday`</td>
+      <td style="font-family:monospace">2023-12-**16T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`saturday at '01:23:45'`</td>
+      <td style="font-family:monospace">2023-12-**16T01:23:45.000**Z</td>
+    </tr>
+    <tr>
+      <td>`sunday`</td>
+      <td style="font-family:monospace">2023-12-**17T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`sunday at '01:23:45'`</td>
+      <td style="font-family:monospace">2023-12-**17T01:23:45.000**Z</td>
+    </tr>
+    <tr>
+      <td>`this hour`</td>
+      <td style="font-family:monospace">2023-12-18T**12:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`last hour`</td>
+      <td style="font-family:monospace">2023-12-18T**11:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`this week`</td>
+      <td style="font-family:monospace">2023-12-**18T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`last week`</td>
+      <td style="font-family:monospace">2023-12-**11T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`this month`</td>
+      <td style="font-family:monospace">2023-12-**01T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`last month`</td>
+      <td style="font-family:monospace">2023-**11-01T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`this quarter`</td>
+      <td style="font-family:monospace">2023-**10-01T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`last quarter`</td>
+      <td style="font-family:monospace">2023-**07-01T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`this year`</td>
+      <td style="font-family:monospace">2023-**01-01T00:00:00.000**Z</td>
+    </tr>
+    <tr>
+      <td>`last year`</td>
+      <td style="font-family:monospace">**2022-01-01T00:00:00.000**Z</td>
+    </tr>
+    </tbody>
+  </table>
+</Collapser>
+
+### Use a DateTime string
+A time range time can be specified using a DateTime string.
+The parser generally supports the ISO 8601 DateTime format: `'2023-12-18T12:34:56.789-06:00'`.
+But, the DateTime parser can recognize variants of the ISO standard format, like `'2023-12-18 12:34'`.
+Date only (`'2023-12-18'`) strings are supported, resolving to midnight on the given day.
+And, time only (`'12:34:56'`) strings are supported, resolving to the given time on the current day.
```

Comment:
Date-only, time-only

Reply:
Will a time-only string resolve to the given time on the current day if it's in the future?

Reply:
I'm changing those two sentences too "Date-only (`'2023-12-18'`) and time-only (`'12:34:56'`) strings are also supported." since the detailed descriptions further down cover the "time in the future" scenario.